src/test_driver/happy/tests/standalone/inet/BUILD.gn

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("${chip_root}/gn/chip/happy_test.gni")
+
+happy_test("inet_tests") {
+    deps = [
```

Comment:
data_deps?

Reply:
Also prefer to depend on only the targets actually used, not the whole group

data_deps = [
  "${chip_root}/src/inet/tests:TestInetLayerMulticast"
]